apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -81,8 +94,43 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     assertCanCreateQuestion(res.locals);
+    if (req.query?.qid) {
+      const qidFull = `__drafts__/${req.query?.qid}`;
```

Comment:
we assume that all drafts questions start with `__drafts__` (by definition, they do!) so we keep that portion out of the query id and append it on.

Reply:
Rather than all this messing around with QIDs, why not just use the question ID?

Reply:
This change was ultimately made.